docsrc/source/installation.rst

```diff
@@ -69,15 +68,41 @@ Download and installation instruction for the MATLAB environment are provided in
 Installing from source
 ----------------------
 
-To install DeerLab from the source, downloaded or clone the source code from the `DeerLab repository <https://github.com/JeschkeLab/DeerLab>`_.
-DeerLab and its dependencies can be installed by running the following command on a terminal window to install DeerLab as a static Python package::
+If you wish to contribute to DeerLab or like to get the latest updates as they come, install DeerLab from ``git``. If your OS has not ``git`` installed, you can download it and install it from the `official Git distribution <https://git-scm.com/>`_.
+To download (clone) the repository, execute the following from the command line::
 
-		python -m setup.py install
+		git clone git@github.com:JeschkeLab/DeerLab.git
 
+To update to the latest version, go the local directory where DeerLab has been downloaded and execute::
+		
+		git pull origin main 
 
-For developers, in order to install DeerLab but be able to frequently edit the code without having to re-install the package every time, use the command::
+Now in the DeerLab directory run the installation script as follows to install DeerLab:: 
 
-		python -m setup.py develop
+		python -m setup.py install
 
+In order to install DeerLab but be able to edit the code or update frequently without having to re-install the package, use the command::
+
+		python -m setup.py develop
 
 Any changes made to the source code will then immediate effect.
+
+
+Installing from Anaconda
+-------------------------
+
+DeerLab is currently not distributed via the ``conda`` or Anaconda package managers. However, ``conda`` or Anaconda users can still install and/or update the software as follows. 
+
+From a terminal (preferably with administrative privileges) choose a local directory where the program should be stored and clone DeerLab from the source as described above :: 
+
+		git clone git@github.com:JeschkeLab/DeerLab.git

```

Comment:
Replace clone statement with:
`git clone https://github.com/JeschkeLab/DeerLab.git`

I‘m not sure how to handle the statement above for contributors.

Reply:
Good point! We need to change this!